nanoFramework.Tools.DebugLibrary.Shared/Runtime/RuntimeValue_Var.cs

```diff
@@ -0,0 +1,36 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// Portions Copyright (c) Microsoft Corporation.  All rights reserved.
+// See LICENSE file in the project root for full license information.
+//
+
+namespace nanoFramework.Tools.Debugger
+{
+    public class RuntimeValue_Var : RuntimeValue
+    {
+        protected internal RuntimeValue_Var(Engine eng, WireProtocol.Commands.Debugging_Value handle) : base(eng, handle)
+        {
+        }
+
+        public override bool IsReference { get { return false; } }
+        public override bool IsNull { get { return false; } }
+        public override bool IsPrimitive { get { return false; } }
+        public override bool IsValueType { get { return false; } }
+        public override bool IsArray { get { return false; } }
+        public override bool IsReflection { get { return false; } }
+        public override bool IsGenericInst { get { return false; } }
+
+        //public override uint NumOfFields
```

Comment:
A bit more dead code here as well

Reply:
Not needed at this time. Still issues with displaying generic objects...